boefjes/boefjes/plugins/kat_sec_txt/boefje.json

```diff
@@ -0,0 +1,14 @@
+{
+    "id": "security_txt",
+    "name": "Security.txt",
+    "description": "check",
```

Comment:
A better description here would be useful:

![image](https://github.com/minvws/nl-kat-coordination/assets/20283/416fb62b-1934-467b-be4a-f76cda8ba6a7)